django/core/checks/security/base.py

```diff
@@ -196,21 +210,36 @@ def check_ssl_redirect(app_configs, **kwargs):
     return [] if passed_check else [W008]
 
 
+def _check_secret_key(secret_key):
+    return (
+        len(set(secret_key)) >= SECRET_KEY_MIN_UNIQUE_CHARACTERS and
+        len(secret_key) >= SECRET_KEY_MIN_LENGTH and
+        not secret_key.startswith(SECRET_KEY_INSECURE_PREFIX)
+    )
+
+
 @register(Tags.security, deploy=True)
 def check_secret_key(app_configs, **kwargs):
     try:
         secret_key = settings.SECRET_KEY
     except (ImproperlyConfigured, AttributeError):
         passed_check = False
     else:
-        passed_check = (
-            len(set(secret_key)) >= SECRET_KEY_MIN_UNIQUE_CHARACTERS and
-            len(secret_key) >= SECRET_KEY_MIN_LENGTH and
-            not secret_key.startswith(SECRET_KEY_INSECURE_PREFIX)
-        )
+        passed_check = _check_secret_key(secret_key)
     return [] if passed_check else [W009]
 
 
+@register(Tags.security, deploy=True)
+def check_secret_key_fallbacks(app_configs, **kwargs):
+    try:
+        fallbacks = settings.SECRET_KEY_FALLBACKS
+    except (ImproperlyConfigured, AttributeError):
+        passed_check = False
+    else:
+        passed_check = all(_check_secret_key(key) for key in fallbacks)
```

Comment:
Just checking - this doesn't account for accidentally passing in a _str_ by say, omitting the comma of a _tuple_. Is an error suitably raised earlier/later by it being a `tuple_settings`? (I may have asked this on the previous PR, I just cannot recall)

Reply:
> Just checking - this doesn't account for accidentally passing in a _str_ by say, omitting the comma of a _tuple_. Is an error suitably raised earlier/later by it being a `tuple_settings`?

Yes exactly, `tuple_settings` protects from passing a string.